packages/aws-client/src/StepFunctions.ts

```diff
@@ -48,16 +61,14 @@ export const describeExecution = improveStackTrace(
  * If a ThrottlingException is received, this function will retry using an
  * exponential backoff.
  *
- * @param {Object} params
- * @returns {Promise<Object>}
+ * @param {DescribeStateMachineInput} params
+ * @returns {Promise<DescribeStateMachineOutput>} DescribeStateMachineOutput
```

Comment:
Same as above re: docstrings. Take it or leave it.

Reply:
noted, I removed the type name from the return description